src/states/node.rs

```diff
@@ -1579,12 +1579,12 @@ impl Node {
                            &self.full_id)
     }
 
-    /// Returns whether we are the recipient of a message for the given authority.
-    fn is_recipient(&self, dst: &Authority) -> bool {
+    /// Returns whether we are a part of the given authority.
+    fn in_authority(&self, dst: &Authority) -> bool {
```

Comment:
Maybe it would be a good idea to rename the parameter - `dst` played nicely with `is_recipient`, but it sounds a bit strange with `in_authority`.